src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --color-hover: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
It's recommended to add fallback fonts as a safety measure. In case the main font doesn't load, the fallback font will be used. This will ensure that your website is always displayed as intended. You can add a generic font family as a fallback after Roboto, such as Arial or sans-serif.